### DISCUSSION TITLE
特定のキーが動かない

### DISCUSSION THREAD

**Discussion by kawamura0807** [Yowkees/keyball#231] [Q&A]
過去の質問も拝見しましたが、解決しなかったため質問いたします。
ball sideの特定のキー（写真で左から２つ、上から３つ目の場所です）のキーのみ反応しません。
関係するダイオードを特定して、写真のようにピンセットで短絡させた上でキーを押すと反応するのですが、ピンセットを外すと反応しなくなります。
目視では、ダイオードとキーソケットの両方ともはんだで固定されているようです。
お忙しい中申し訳ございませんが、ご回答いただけると幸いです。

![IMG_5191](https://github.com/Yowkees/keyball/assets/110145678/1e37fde5-bd97-463e-9127-6fa9e3998a7d)

**Comment by Yowkees** [comment]
![KB44R_fix](https://github.com/Yowkees/keyball/assets/84209772/4570ffc7-54bc-4604-8387-47e78533782e)

ご連絡ありがとうございます。
基板データを確認してみましたところ、左から２つ、上から３つ目のキーに該当するダイオードは緑丸で示したものと思います。
緑丸のダイオードの両端子がテンプラ（半田付け不良）していないかご確認お願いいたします。
